### PR TITLE
[17.0][MIG] partner_sale_pivot

### PR DESCRIPTION
**Pull request by Wodran14**

### CODE REVIEW

**Review by flotho — APPROVED** [review]
functionnal review on runboat
LGTM

File: partner_sale_pivot/readme/CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- [Tecnativa](https://www.tecnativa.com):
+  - Ernesto Tejeda
+  - Pedro M. Baeza
+  - Sergio Teruel
```
Comment (rousseldenis) [inline partner_sale_pivot/readme/CONTRIBUTORS.md:4]: @Wodran14 FYI, when doing a migration, you can add yourself to contributors :smiley:

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**